packages/hdwallet-vultisig/src/bitcoin.ts

```diff
@@ -0,0 +1,172 @@
+import * as bitcoin from "@shapeshiftoss/bitcoinjs-lib";
+import * as core from "@shapeshiftoss/hdwallet-core";
+import { BTCInputScriptType } from "@shapeshiftoss/hdwallet-core";
+
+import { VultisigUtxoProvider } from "./types";
+
+const getNetwork = (coin: string): bitcoin.networks.Network => {
+  switch (coin.toLowerCase()) {
+    case "bitcoin":
+      return bitcoin.networks.bitcoin;
+    default:
+      throw new Error(`Unsupported coin: ${coin}`);
+  }
+};
+
+export function translateCoin(coin: core.Coin): string {
+  const coinMap: Record<core.Coin, string> = {
+    Bitcoin: "btc",
+    // Litecoin: "ltc",
+    // Zcash: "zec",
+    // BitcoinCash: "bch",
+    // Dash: "dash",
+    // Dogecoin: "doge",
+  };
+  return core.mustBeDefined(coinMap[coin]);
+}
+
+export type BtcAccount = string;
+
+export const btcGetAccountPaths = (msg: core.BTCGetAccountPaths): Array<core.BTCAccountPath> => {
+  const slip44 = core.slip44ByCoin(msg.coin);
+  if (slip44 === undefined) return [];
+
+  const bip84 = core.segwitNativeAccount(msg.coin, slip44, msg.accountIdx);
+
+  const coinPaths = {
+    bitcoin: [bip84],
```

Comment:
confirming only segwit native support for utxo chains

Reply:
Looks like this was copypasted from Phantom (SegWit native support only) but yeah looking at vultisig codebase, not sure if only SegWit native is supported.
Definitely wouldn't hurt to have support explicitly commented here